### PR TITLE
fix: disable buttons behind dialog

### PR DESCRIPTION
**Pull request by gkatsev**
Alternative to https://github.com/muxinc/elements/pull/427

### CODE REVIEW

File: packages/mux-player/src/media-theme-mux/media-theme-mux.ts [hunk 1]
```diff
@@ -75,7 +79,7 @@ export default class MediaThemeMux extends MediaTheme {
     };
 
     // if we don't have a src, we want to disable all hotkeys
-    props.nohotkeys = props.hasSrc ? props.nohotkeys : true;
+    props.nohotkeys = !props.hasSrc || props.isDialogOpen ? true : props.nohotkeys;
```
Comment (cjpillsbury) [inline packages/mux-player/src/media-theme-mux/media-theme-mux.ts:82]: nitpick(non-blocking): `props.nohotkeys || !props.hasSrc || props.isDialogOpen`

File: packages/mux-player/src/template.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ const isLiveOrDVR = (props: MuxTemplateProps) =>
 export const content = (props: MuxTemplateProps) => html`
   <${unsafeStatic(castThemeName(props.theme) ?? 'media-theme-mux')}
     has-src="${props.hasSrc}"
+    has-open-dialog="${props.isDialogOpen}"
```
Comment (cjpillsbury) [inline packages/mux-player/src/template.ts:32]: thought(non-blocking): We'll want to be thinking about how to handle these scenarios in a theme-agnostic way going forward.

File: packages/mux-player/src/media-theme-mux/media-theme-mux.ts (hunk 1 above)
Comment (gkatsev) [reply packages/mux-player/src/media-theme-mux/media-theme-mux.ts:82]: In the case where a source isn't set, or the dialog is open, we want to override the user-provided value of `nohotkeys`.

Comment (cjpillsbury) [reply packages/mux-player/src/media-theme-mux/media-theme-mux.ts:82]: only if it's `false`, no?

**Review by cjpillsbury — APPROVED** [review]
approved with a couple of non-blocking comments/callouts.

File: packages/mux-player/src/template.ts (hunk 2 above)
Comment (gkatsev) [reply packages/mux-player/src/template.ts:32]: Yeah, as I mentioned, https://github.com/muxinc/elements/pull/427 is theoretically a more correct fix and would be theme agnostic. However, that change is a bit more complicated. There's potentially a performance issue but there's also an issue around binding the key against the document and trying to find all focusable elements and mux-player, which might not be possible depending on how mux-player is included on the page.

File: packages/mux-player/src/media-theme-mux/media-theme-mux.ts (hunk 1 above)
Comment (gkatsev) [reply packages/mux-player/src/media-theme-mux/media-theme-mux.ts:82]: Ah, right, if nohotkeys is true, it probably doesn't matter what the others are set to. Was overthinking things.
Yeah, your suggestion is better.

**Pull request merged**